opendebates_emails/__init__.py

```diff
@@ -0,0 +1 @@
+#
```

Comment:
Is there a better way to add an empty file into version control?  This is how I've always done it (a blank file with just an empty comment)